tests/utils/command_line_params.h

```diff
@@ -233,7 +355,9 @@ extern bool verify_results;
 extern bool low_mode_check;
 extern bool oblique_proj_check;
 extern double mass;
+extern quda::mass_array<double> mass_array;
```

Comment:
I think `mass_array` and `kappa_array`, along with the corresponding options to set it on the command line, can be deleted, purely because they aren't being used anywhere. They're also somewhat redundant of `multishift_masses` in some regard... that said, I do like the logic in `add_massoption`, so maybe we want to make a note of it somewhere and apply it to `multishift_masses`, etc.

Reply:
I see `kappa_array` in `tests/host_reference/domain_wall_dslash_reference.cpp`. `mass_array` is indeed an artifact, will remove.

Reply:
fixed in 88a72635edcfd2a4e36f6bd9b9b71a6d34a59cc3